cfme/physical/physical_server.py

```diff
@@ -105,21 +117,33 @@ def refresh(self, cancel=False):
         view.toolbar.configuration.item_select("Refresh Relationships and Power States",
             handle_alert=cancel)
 
-    def wait_for_physical_server_state_change(self, desired_state, timeout=300):
+    def wait_for_state_change(self, desired_state, target, provider, timeout=300, delay=10):
```

Comment:
**REQUIRED**
This method should be private and called in other methods such as `power_off` and `power_on`:
```python
def power_on(self, wait=True):
    if wait:
        self._wait_for_state_change(...)
    ...
```

Reply:
done